nix/modules/toml-mapping.nix

```
@@ -32,5 +49,14 @@ in
     kuutamo.network.ipv6.address = cfg.ipv6_address or null;
     kuutamo.network.ipv6.gateway = cfg.ipv6_gateway or null;
     kuutamo.network.ipv6.cidr = cfg.ipv6_cidr or 128;
+
+    kuutamo.telegraf.url = monitor_cfg.url;
+    kuutamo.telegraf.username = monitor_cfg.username;
+    kuutamo.telegraf.password = monitor_cfg.password;
+
+    kuutamo.telegraf.kmonitoring_url = kmonitor_cfg.url;
+    kuutamo.telegraf.kmonitoring_protocol = kmonitor_cfg.protocol;
+    kuutamo.telegraf.kmonitoring_user_id = kmonitor_cfg.user_id;
+    kuutamo.telegraf.kmonitoring_password = kmonitor_cfg.password;
```

You could nest these objects in rust for expamle in the config struct:
```suggestion
    kuutamo.telegraf = cfg.telegraf;

    kuutamo.hosted_monitoring = cfg.hosted_monitoring;
```

---

These parameters are separated intentionally.  Because when `kneard-mgr generate-config /tmp/kneard`, the monitor secrets will store separately,  and try to avoid possible leakage when the user sends out the configuration files for any technical support.